src/output.bash

```diff
@@ -38,7 +38,7 @@ batslib_err() {
   { if (( $# > 0 )); then
       echo "$@"
     else
-      ( PATH="$BATS_SAVED_PATH"; command cat -; )
+      PATH="$BATS_SAVED_PATH" command cat -
```

Comment:
Does that actually work if `cat` was mocked? Just want to make sure and can't test right now.

Reply:
Isn't there a test for that?

Reply:
I guess it works due to the use of `command`. I was just wondering if the change from `PATH=...; command cat` to `PATH=... command cat` made the PATH modification void.